chef/cookbooks/hyperv/recipes/openstack_install.rb

```diff
@@ -41,7 +41,7 @@
 
 windows_batch "unzip_neutron" do
   code <<-EOH
-  #{node[:sevenzip][:command]} x #{node[:cache_location]}#{node[:openstack][:neutron][:file]} -o#{node[:openstack][:location]}\\dist -r -y
+  #{node[:sevenzip][:command]} x #{node[:cache_location]}#{node[:openstack][:neutron][:file]} -o#{node[:openstack][:location]} -r -y
   #{node[:sevenzip][:command]} x #{node[:openstack][:location]}\\dist\\#{node[:openstack][:neutron][:name]}-#{node[:openstack][:neutron][:version]}.tar -o#{node[:openstack][:location]} -r -y
   rmdir /S /Q #{node[:openstack][:location]}\\dist
```

Comment:
I'm unclear how this could be unpacked in /dist/dist? Why is that the case for neutron, but not for nova (a few lines above)?